### DISCUSSION TITLE
增加倍速调节功能

### DISCUSSION THREAD

**Discussion by Mishasama** [apades/dmMiniPlayer#19] [Ideas]
由于 API 不同，所以原有能控制 PiP 播放速度的方式都失效了，只能先调节播放速度后再激活这个新的 PiP 模式。

希望能给个控制方式，大约在`0.1x ~ 16.0x`的范围？或者无上限？

**Comment by apades** [comment]
> 由于 API 不同，所以原有能控制 PiP 播放速度的方式都失效了

什么原有的播放速度方式啊，我试了我这个旧API的开启倍速功能还在的

![image](https://github.com/user-attachments/assets/9489a4e6-e986-4835-a380-cdf75692ea7a)

**Reply by Mishasama** [reply]
我指的是网站原来的播放器自带的倍速播放调节。

**Comment by apades** [comment]
如果你想说在播放器里加个`倍速功能`，这个最近应该会加上，我最近长按倍速有时候要按挺久的难受到我了😫

**Reply by Mishasama** [reply]
知道就好😏
